bookshelf/1-cloud-run/pom.xml

```diff
@@ -22,11 +22,11 @@ Copyright 2019 Google LLC
   <artifactId>bookshelf-cloud-run</artifactId>
   <version>1.0-SNAPSHOT</version>
 
-  <parent>
-    <groupId>com.google.cloud.samples</groupId>
-    <artifactId>shared-configuration</artifactId>
-    <version>1.0.11</version>
-    <relativePath></relativePath>
+  <parent> <!-- Only used for testing - NOT REQUIRED -->
```

Comment:
We use the `com.google.cloud.samples.shared-configuration` as the parent. This is what we use for checkstyle testing.

Reply:
That's actually included up the chain (in the pom.xml in the root of the repository)